docker/docker-compose.yml

```
@@ -19,10 +19,15 @@ x-php: &php
     - "s3:s3.localhost"
   external_links:
     - "proxy:${COMPOSE_PROJECT_NAME:-default}.altis.dev"
+    - "proxy:pinpoint-${COMPOSE_PROJECT_NAME:-default}.altis.dev"
+    - "proxy:cognito-${COMPOSE_PROJECT_NAME:-default}.altis.dev"
```

If we're adding these we should explicitly add `elasticsearch` and `s3` too
```suggestion
    - "proxy:cognito-${COMPOSE_PROJECT_NAME:-default}.altis.dev"
    - "proxy:elasticsearch-${COMPOSE_PROJECT_NAME:-default}.altis.dev"
    - "proxy:s3-${COMPOSE_PROJECT_NAME:-default}.altis.dev"
```